index.js

```diff
@@ -195,7 +200,24 @@ function buildCache (client, opts) {
       results.forEach((key) => multi.get(key));
       return asPromise(multi.exec.bind(multi));
     })
-    .then((results) => results.map(JSON.parse));
+    .then((results) => {
+      return results.map((res) => {
+        if (Array.isArray(res)) {
+          return JsonParse(res[1]);
+        } else {
+          return JsonParse(res);
+        }
+      });
+    });
+
+  const JsonParse = (str) => {
+    try {
```

Comment:
console.error is probably not the way to go for a library. Why wouldn't you throw an error if the parsing fails?